MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -83,6 +88,22 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func getCurrentPosition(callbackId: String) {
+
+        self.executeJavaScriptCallback(responseStatus: .onSuccess, messageId: callbackId, response: getLocationInfo())
+    }
+
+    func getLocationInfo() -> String {
+        return  "{\"coords\":{\"latitude\":\(locationManager?.location?.coordinate.latitude ?? 0)" +
+        ",\"longitude\":\(locationManager?.location?.coordinate.longitude ?? 0)" +
+        ", \"altitude\":\(locationManager?.location?.altitude ?? 0)" +
+        ", \"altitudeAccuracy\":\(locationManager?.location?.verticalAccuracy ?? 0)" +
+        ", \"accuracy\":\(locationManager?.location?.horizontalAccuracy ?? 0)" +
+        ", \"speed\":\(locationManager?.location?.speed ?? 0)" +
+        ", \"heading\":\(locationManager?.location?.course ?? 0)" +
+        "}, \"timestamp\":\(Date().currentDate)}"
```

Comment:
for more clarity, you can write multiple lines strings with triple quotes `"""` :
```swift
return """
        {
            "coords": {
                "latitude":\(locationManager?.location?.coordinate.latitude ?? 0),
                "longitude":\(locationManager?.location?.coordinate.longitude ?? 0),
                "altitude":\(locationManager?.location?.altitude ?? 0),
                "altitudeAccuracy":\(locationManager?.location?.verticalAccuracy ?? 0),
                "accuracy":\(locationManager?.location?.horizontalAccuracy ?? 0),
                "speed":\(locationManager?.location?.speed ?? 0),
                "heading":\(locationManager?.location?.course ?? 0)
            },
            "timestamp":\(Date().currentDate)
        }
        """
```

Reply:
oh my bad, I guess you just want a single line quote

Reply:
That's right. Multi-line string won't work here